specification/programmableconnectivity/Azure.ProgrammableConnectivity/apis/common.tsp

```diff
@@ -0,0 +1,110 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.ProgrammableConnectivity;
+
+alias ServiceTraits = NoRepeatableRequests &
+  NoConditionalRequests &
+  SupportsClientRequestId &
+  RequestHeadersTrait<ApcGatewayIdHeader>;
+
+alias Operations = Azure.Core.ResourceOperations<
+  ServiceTraits,
+  ApcErrorResponse
```

Comment:
The shape of the error model is not conformable to guideline https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#rest-error-response-body-structure

Is it part of "CAMARA-like"?